sheet.go

```diff
@@ -110,9 +182,15 @@ func (s *Sheet) makeXLSXSheet(refTable *RefTable, styles *xlsxStyleSheet) *xlsxW
 	maxRow := 0
 	maxCell := 0
 
+	// Scan through the sheet and see if there are any merged cells. If there
+	// are, we may need to extend the size of the sheet. There needs to be
+	// phantom cells underlying the area covered by the merged cell
```

Comment:
I guess we think about this differently. To me the "creating cells" is just a side-effect. They might very well already be created by the user, for example. To me `handleMerged` isn't a cell-creating function, it's a border adjusting one. It makes sure that all cells included by the merge has the correct border.

But to actually answer your quesiton, maybe I think it's slightly superfluous in the fact that it's describing (partially) the behavior of another function, e.g. `handleMerged`. But reading it again I think mentioning the border adjusting there would be sufficient.

I don't feel strongly about this though, you can leave it as-is if you want.